data/actions/scripts/other/create_bread.lua

```diff
@@ -4,7 +4,7 @@ local millstones = {1381, 1382, 1383, 1384}
 function onUse(player, item, fromPosition, target, toPosition, isHotkey)
 	local itemId = item:getId()
 	if itemId == 2692 then
-		if target.type == 1 and table.contains(liquidContainers, target:getId()) then
+		if target:getSubType() == 1 and table.contains(liquidContainers, target:getId()) then
```

Comment:
I think somehow, .type is sometimes used as getCount?

Reply:
target:getSubType() does not work, correct is getCount()